src/app/shared/core/series-card/series-card.component.ts

```diff
@@ -0,0 +1,8 @@
+import { Component } from '@angular/core';
+
+@Component({
+  selector: 'app-series-card',
+  templateUrl: './series-card.component.html',
+  styleUrls: ['./series-card.component.scss'],
+})
+export class SeriesCardComponent {}
```

Comment:
Because you define `SeriesCardComponent` in `shared/core`, you should declare it in `SharedModule` instead of separate feature modules, such as `HomeModule`.